using_images/s2i_images/dot_net_core.adoc

```diff
@@ -0,0 +1,71 @@
+[[using-images-using-dot-net-core]]
+= .NET Core
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[benefits-of-using-dot-net-core]]
+== Benefits of Using .NET Core
+
+{nbsp}.NET Core is a general purpose development platform featuring automatic memory
+management and modern programming languages. It allows users to build high
+quality applications efficiently. .NET Core is available on Red Hat Enterprise
+Linux (RHEL 7) and {product-title} via certified containers. .NET Core offers:
+
+* The ability to follow a microservices-based approach, where some components are
+built with .NET and others with Java, but all can run on a common, supported
+platform in Red Hat Enterprise Linux and {product-title}.
+* The capacity to more easily develop new .NET Core workloads on Windows;
+customers are able to deploy and run on either Red Hat Enterprise Linux or
+Windows Server.
+* A heterogeneous data center, where the underlying infrastructure is capable of
+running .NET applications without having to rely solely on Windows Server.
+* Access to many of the popular development frameworks such as .NET, Java, Ruby,
+and Python from within {product-title}.
+
+[[dot-net-core-supported-versions]]
+== Supported Versions
+
+* .NET Core framework version 1.0.
+* .NET is supported by Red Hat Enterprise Linux (RHEL) 7.
+* {product-title} versions 3.3 and later are supported.
+
+[[dot-net-core-installing-images]]
+== Images
+
+Image stream definitions for the .NET Core on RHEL S2I image are now added
+during {product-title} installations.
+
+[[dot-net-core-configuration]]
+== Configuration
+
+The .NET Core image currently does not support any environment variables. No
+configuration is necessary.
+
+[[dot-net-core-hot-deployment]]
+== Hot Deployment
+
+Hot deployment allows you to quickly make and deploy changes to your application
+without having to generate a new build.
+
+Use the
+xref:../../dev_guide/new_app.adoc#specifying-environment-variables[`oc new-app`]
+command to build a new application.
+
+
+To change your source code in a running pod, use the
+xref:../../cli_reference/basic_cli_operations.adoc#troubleshooting-and-debugging-cli-operations[`oc rsh`]
+command to enter the container:
+
+----
+$ oc rsh <pod_id>
+----
+
+After you enter into the running container, your current directory is set to
+*_/opt/app-root/src_*, where the source code is located.
```

Comment:
this seems cut off?

also do we actually know that the .NET image supports hot deployment?  Since it's a compiled language i'd expect at a minimum that you need to sync compiled files to the container, not just modify live source.